src/app/pages/event-card/event-card.component.scss

```diff
@@ -1,61 +1,38 @@
-@import '../../../assets/config.scss';
```

Comment:
I've updated the code to use bootstrap's standard classes. It's good to have our own style sets, but it would defeat the purpose of using a standard library like bootstrap since it already provides a good card layout.